gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -20,6 +20,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
 
   OrchestrationTemplateCfn.register_eligible_manager(self)
   ExtManagementSystem.register_cloud_discovery_type('amazon' => 'ec2')
+  MiqProvision.register_supported_ems(self)
```

Comment:
Are you saying that maybe there needs to be an initializer for this in the provider's plugin?